src/components/dialogs/AddStaffNotationDialog.vue

```diff
@@ -79,233 +115,272 @@ import Vue from 'vue'
 import { Component, Prop, Watch, Emit } from 'vue-property-decorator'
 import { Getter } from 'vuex-class'
 import { DateMixin, EnumMixin } from '@/mixins'
-import axios from '@/axios-auth'
 import { CourtOrderPoa } from '@bcrs-shared-components/court-order-poa'
+import FileUploadPdf from '@/components/common/FileUploadPdf.vue'
 import { FormIF } from '@/interfaces'
-import { EffectOfOrderTypes, FilingTypes, DissolutionTypes } from '@/enums'
+import { EffectOfOrderTypes, FilingTypes, DissolutionTypes, PageSizes } from '@/enums'
+import { LegalServices } from '@/services'
 
 @Component({
-  components: { CourtOrderPoa },
-  mixins: [
-    DateMixin,
-    EnumMixin
-  ]
+  components: { CourtOrderPoa, FileUploadPdf },
+  mixins: [ DateMixin, EnumMixin ]
 })
 export default class AddStaffNotationDialog extends Vue {
   $refs!: Vue['$refs'] & {
     courtOrderPoaRef: FormIF,
+    fileUploadRef: FormIF,
```

Comment:
The file uploader component provides the same form validation functions as a v-form!
- validate()
- reset()
- resetValidation()